proposals/NNNN-SDL-passenger-mode.md

```diff
@@ -0,0 +1,60 @@
+# SDL Passenger Mode
+
+* Proposal: [SDL-NNNN](NNNN-mobile-api-versioning-revision.md)
```

Comment:
As this will be revisions to a previously withdrawn proposal (0119), I think we should be making changes to that proposal file (0119-SDL-passenger-mode.md) instead of creating a new one.